build.gradle

```diff
@@ -2,11 +2,12 @@
 buildscript {
     repositories {
         maven { url 'https://dl.bintray.com/doppllib/maven2' }
+        maven { url 'https://dl.google.com/dl/android/maven2/' }
```

Comment:
can be `google()`.

Reply:
I didnt use `google()` because i've had issues with doppl when using it in other projects, but it seems that isn't an issue here, though dopplbuild is failing for other reasons. I'll add it. Thanks!